src/kaltura-live-plugin.tsx

```diff
@@ -245,7 +245,9 @@ export class KalturaLivePlugin implements OnMediaUnload, OnPluginDestroy {
 
         if (receivedState === LiveBroadcastStates.Offline) {
             this._manageOfflineSlate(OfflineTypes.NoLongerLive);
-            this._player.dispatchEvent(new (KalturaPlayer.core as any).FakeEvent(this._player.Event.ENDED));
+            if (this._attachMedia) {
```

Comment:
seems like `_attachMedia` related to the dvr. 
the question is dont you need to handle primary to secondary switching for non dvr as well?